compiler/next/include/chpl/resolution/resolution-types.h

```diff
@@ -279,6 +279,13 @@ class CallInfo {
       : name_(name), hasQuestionArg_(hasQuestionArg),
         actuals_(std::move(actuals)) {}
 
+  CallInfo(UniqueString name, bool isMethod, bool hasQuestionArg,
```

Comment:
We could also compute the method-ness by looking for a `this=` argument.

Anyway I would prefer if you delete the other constructor so we can't forget to supply `isMethod`.